.env.local

```diff
@@ -0,0 +1,14 @@
+VITE_API_URL=http://localhost:3000/api
+VITE_APP_MODE=development
+
+VITE_ZITADEL_PROJECT_ID=
```

Comment:
VITE prefix is extra, we should remove it

Reply:
alright ill make the changes